test/performance/comm/barrier/empty-chpl-barrier.chpl

```diff
@@ -0,0 +1,37 @@
+use Time;
+use Barriers;
+
+config const numTrials = 100;
+config const printTimings = false;
+
+config const numTasksPerLocale = here.maxTaskPar;
+const numTasks = numLocales * numTasksPerLocale;
+
+enum BarrierMode {
+  LocalAtomic
+};
+use BarrierMode;
+
+config param barrierMode = LocalAtomic;
+
+proc main() {
+  var t: Timer;
+
+  t.start();
+  select barrierMode {
+    when LocalAtomic do LocalBarrierBarrier();
```

Comment:
This can be a TODO, but I'd slightly prefer that we use ``barrierMode`` to create the barrier, and then pass it to some common function as an argument.

Reply:
That gets tricky in cases I want to add where we have 2 barriers -- One per-locale and 1 between locales, so we only have to do numLocales on-stmts instead of numLocales * numTasksPerLocales.